tidb-cloud/data-service-manage-endpoint.md

```diff
@@ -62,14 +109,23 @@ On the right pane of the endpoint details page, you can click the **Properties**
     - Minimum value: `1`
     - Unit: millisecond
 
-- **Max Rows**: the maximum number of rows that the endpoint returns.
+- **Max Rows**: the maximum number of rows that the endpoint can operate or return.
 
     - Default value: `50`
-    - Maximum value: `2000`
+    - Maximum value: `2000` when **Batch Operation** is disabled and `100` when **Batch Operation** is enabled for an endpoint
     - Minimum value: `1`
 
 - **Description** (Optional): the description of the endpoint.
 
+#### Advanced properties
+
+- **Tag**: the tag used for identifying a group of endpoints.
+- **Batch Operation**: this property is visible only when the request method is `POST`, `PUT`, or `DELETE`. When **Batch Operation** is enabled, you can operate on multiple rows in a single request. For example, you can insert multiple rows of data in a single `POST` request by adding an array of data objects in the `--data-raw` option of your curl command when calling the endpoint.
+
+    > **Note:**
+    >
+    > If the request method of your endpoint is `DELETE` and **Batch Operation** is enabled, you need to use `%2C` instead of comma (`,`) to separate the rows to be deleted in curl code, such as `/endpoint/<Endpoint Path>?id=${id}%2C${id}`. This is because comma (`,`) is a reserved character in URLs and cannot be used directly.
```

Comment:
```suggestion
    > If the request method of your endpoint is `DELETE` and **Batch Operation** is enabled, you need to use `%2C` instead of comma (`,`) to separate the rows to be deleted in your curl command, such as `/endpoint/<Endpoint Path>?id=${id}%2C${id}`. This is because comma (`,`) is a reserved character in URLs and cannot be used directly.
```